dockerfiles/examples/ml/build.sh

```diff
@@ -19,30 +19,54 @@
 # Use this script to build local images for Machine Learning HPCC Systems Docker images
 #
 
+usage()
+{
+  echo "Usage: build.sh [options]"
+  echo "    -h     Display help"
+  echo "    -l     Tag the images as the latest"
+  echo "    -m     ML feature: one of scikit, tf2 and tf2-gpu"
+  echo "    -t     Tag of base image hpccsystems/platform-core"
+  exit
+}
 LABEL=
-while getopts “hlt:” opt; do
+FEATURE=
+while getopts “hlm:t:” opt; do
   case $opt in
     l) TAGLATEST=1 ;;
+    m) FEATURE=$OPTARG ;;
     t) LABEL=$OPTARG ;;
-    h) echo "Usage: incr.sh [options]"
-       echo "    -h                 Display help"
-       echo "    -l                 Tag the images as the latest"
-       echo "    -t                 Tag of base image hpccsystems/platform-core"
-       exit
-       ;;
+    h) usage   ;;
   esac
 done
 shift $(( $OPTIND-1 ))
 
-[[ -z ${LABEL} ]] && LABEL=latest
-
+[[ -z ${FEATURE} ]] && usage
 
 ml_features=(
   'scikit'
-  #'tf2'
-  #'tf2-gpu'
+  'tf2'
+  'tf2-gpu'
 )
 
+found="false"
+for ml_feature in ${ml_features[@]}
+do
+	if [[ $ml_feature ==  $FEATURE ]]
+	then
+          found="true"
+	  break
```

Comment:
indentation a little odd, I think there are tabs here (that should be replaced by spaces)

Reply:
fixed